pom.xml

```diff
@@ -67,6 +67,7 @@
         <jqassistant.build.version>2.6.2</jqassistant.build.version>
         <!-- library versions -->
         <assertj.version>3.27.3</assertj.version>
+        <xmlunit-assertj.version>2.10.0</xmlunit-assertj.version>
```

Comment:
The version properties should be in alphabetical order (even if I understand the intention to add it here)